src/Gui/QSint/actionpanel/taskheader_p.cpp

```diff
@@ -111,11 +111,10 @@ bool TaskHeader::eventFilter(QObject *obj, QEvent *event)
 }
 
 void TaskHeader::setScheme(ActionPanelScheme *scheme)
-{
+{  
```

Comment:
is the whitespace necessary here?

Reply:
Thank you, i didn't notice that. VScode diff did not show whitespace-only lines to me. I removed the two erroneous whitespaces again.